DSharpPlus.VoiceNext/EventArgs/VoiceReceiveEventArgs.cs

```diff
@@ -14,10 +14,14 @@ public class VoiceReceiveEventArgs : DiscordEventArgs
         /// </summary>
         public uint SSRC { get; internal set; }
 
+        #nullable enable
```

Comment:
Ehhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh. I very much dislike the nullability feature because all it does is cause compiler noise, without really assisting the user. Not that most people read those, but xmldoc note might be far less annoying.